maxmin/max2/max2.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+/* (maxmin/max2) a program that calculates the maximum of two numbers. You can use the following formula: max(a, b) = (a + b + |a - b|)/2. You will need to use math.Abs function to calculate an absolute value. Example:
+
+The program finds the maximum of two numbers max(a, b).
+Enter two numbers:
+5 7
+max: 7 */
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("The program finds the maximum of two numbers.")
+	fmt.Println("Enter two numbers:")
+	var a float64
+	var b float64
+	fmt.Scan(&a, &b)
+	var max = math.Max(a, b)
```

Comment:
Please avoid using `math.Max`. This exercise was about using the fact that `max(a, b)` is the same as `(a + b + |a - b|) / 2`

Reply:
The same goes for max3, min2 and min3 programs.